docs/migration/Version_0.3.4_0.4.0 - Guide/3-policy-definitions.md

```diff
@@ -0,0 +1,98 @@
+# Creating a Policy Definition
+
+### Old plain JSON Schema
+
+```json
+{
+  "id": "policy-definition-id",
+  "policy": {
+    "permissions": [
+      {
+        "action": {
+          "type": "USE"
+        },
+        "constraints": [
+          {
+            "leftExpression": {
+              "value": "BusinessPartnerNumber"
+            },
+            "rightExpression": {
+              "value": "BPN"
```

Comment:
I would not just use the term `BPN`. Someone who does not have much experience with EDC might think that it is a constant and that this field must always be filled with exactly this value. Therefor I would suggest something like this:

```suggestion
              "value": "ANY-BPN"
```

or even more concrete:

```suggestion
              "value": "BPN00000000"
```